### ISSUE TITLE
Purify not working after Meteor 2.5 upgrade

### ISSUE THREAD

**Issue by dotmlj** [vazco/meteor-universe-i18n#149]
We are a long time user of universe:i18n in our Meteor React projects (thanks!) and recently upgraded a project from Meteor 1.11.1 to 2.5.3, which also upgrading universe:i18n from 1.31.0 to 1.32.5.

After the upgrade any i18n strings that include HTML, shows the HTML as a string instead of rendering it. Something `_purify: false` previously fixed for us.

```
const T = i18n.createComponent();
...

<Typography variant="body1">
   <T _purify={false}>dashboard.support_info</T>
</Typography>
```

Using `dangerouslySetInnerHTML` renders the same result
```
<Typography
    variant="body1"
    dangerouslySetInnerHTML={{
        __html: i18n.__('dashboard.support_info', {
        _purify: false,
        }),
    }}
/>
```

The `dashboard.support_info` string in our `en.i18n.json` file:
`"support_info": "Write to us at <a href='mailto:hi@email.com'>hi@email.com</a>",`

Any ideas are highly appreciated :)

**Comment by dotmlj** [comment]
After having downgrade universe:i18n one version at a time, it seems that this problem started after the version 1.32.0 update, so changing to version 1.31.0 fixed the problem for me.

`meteor add universe:i18n@=1.31.0`

It would of course still be great to find the exact problem, to get the improvements made since 1.31.0.

**Comment by i-talic** [comment]
I solved this by adding this code (I'm using Blaze though):

```
i18n.setOptions({
  purify: string => string
});
```

**Comment by piotrpospiech** [comment]
Hi @dotmlj, sorry for the late reply. I reproduced your issue. It seems like we changed the `_purify` property to `purify` when we were moving this project to Typescript (`v1.32`). 

This code should work:
```js
<T purify={false}>dashboard.support_info</T>
```

We will update the documentation soon.

**Comment by radekmie** [comment]
@piotrpospiech Are you sure about that? I'd rather change it back, as right now it may interfere with people using `purify` variable in their translations.

**Comment by piotrpospiech** [comment]
After a short discussion with @radekmie, we decided to keep the new code without breaking change, so I published a new version of the package (`v1.32.6`) with `_purify` instead of `purify`. Now it's consistent with the documentation.

Update package and syntax `_purify` will work in your code.

**Comment by dotmlj** [comment]
Great, thanks a lot!
